### PR TITLE
Stringtie

### PR DESCRIPTION
**Pull request by rannick**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
  - [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnafusion/tree/master/.github/CONTRIBUTING.md)
  - [x] If necessary, also make a PR on the nf-core/rnafusion _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- `stringtie` v2.2.1 added to detect splicing events. Not included in `fusion-report` or `fusionInspector` summaries
```
Comment (maxulysse) [inline CHANGELOG.md:10]:
```suggestion
- `stringtie` v2.2.1 added to detect splicing events. Not yet included in `fusion-report` or `fusionInspector` summaries
```
We probably work on that part

Comment (rannick) [reply CHANGELOG.md:10]: The type of output is very different for stringtie: I don't think it can be processed like the other tools

Comment (maxulysse) [reply CHANGELOG.md:10]: :disappointed:

**Review by apeltzer — APPROVED** [review]
Looks great and local tests ran through :-)

**Pull request merged**